networking/src/ApiClient.ts

```diff
@@ -80,6 +113,157 @@ interface SubscriptionValue<T> {
 }
 
 export class ApiClient implements IApiClient {
+
+    async createGameTemplate(
+        id: string,
+        title: string,
+        owner: string,
+        version: number,
+        description: string,
+        domain: string,
+        cluster: string,
+        grade: string,
+        standard: string,
+        phaseOneTime: number,
+        phaseTwoTime: number,
+        imageUrl: string
+    ): Promise<IGameTemplate | null> {
+        console.log("here");
```

Comment:
Probably don't need it or you can make it more informative and leave the console. I think there's `console.info` that we can start using more often for such comments

Reply:
Yeah, this was just left over from tracking down a bug. It's been removed.